src/pages/people/Person.tsx

```diff
@@ -139,7 +139,12 @@ export default function Person(props: PersonProps) {
             e.preventDefault();
             e.stopPropagation();
           }}
-          style={{ textDecoration: 'none', color: 'inherit', cursor: 'pointer' }}
+          style={{
+            flex: '1 0 min(100%, 340px)',
+            textDecoration: 'none',
+            color: 'inherit',
+            cursor: 'pointer'
+          }}
```

Comment:
Instead of using the `style` prop we can instead make a styled component to make this code look a bit cleaner